.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Bug report
+about: 'We really appreciate your effort to provide feedback. Before opening a new
```

Comment:
Are the quotes (') needed?

Reply:
They were added by GitHub when editing the templates, so I presume "Yes" but haven't tried without them.

Reply:
Ok, I get it. It was strange to me that the second template doesn't have them.